Makefile

```diff
@@ -407,6 +407,9 @@ dev.shell.analyticsapi:
 dev.shell.insights:
 	docker compose exec insights env TERM=$(TERM) bash -c 'eval $$(source /edx/app/insights/insights_env; echo PATH="$$PATH";) && /bin/bash'
 
+dev.shell.enterprise_catalog:
```

Comment:
As the service name in `docker-compose.yml` file is `enterprise-catalog` _with hyphen_, this should also be `dev.shell.enterprise-catalog` because the rest of the commands for this service will be like this

```
make dev.up.enterprise-catalog
make dev.logs.enterprise-catalog
```